index.html

```diff
@@ -0,0 +1,1312 @@
+</div>
+</section>
+<section class="Portfolio__About__MySelf"> 
+            <div class="portfolio__card__myself">
+            <h4 class="About__MySelf__header">About Myself</h4>
+            <div class="About__MySelf__Content">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="About__MySelf__layout">
+              <div class="About__MySelf__design">
+                <h3 class="About__MySelf__Connect">LET'S CONNECT</h3>
+              </div>
+              <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+           </div>
+           <div class="resume__tag">
+            <a class="resume__button"><p>Get my resume</p></a>
+           </div>
+           </div>
+       
+<!-- languages portfolio -->
+<div class="languages_container">
+    <div class="languages_heading-container">
+         
+        <div class="heading">Languages</div>
+        <div class="chevron-container">
+            <div class="chevron">
+                <img class="framework__arr" src="img/about_page/arr_d.svg">
+            </div>
+        </div>
+    </div>
+    <div class="languages_heading-container">
+        <div class="language-box">
+            <img class="language-image" src="img/about_page/javascript.png" />
+            <div class="language-name">JavaScript</div>
+        </div>
+        <div class="language-box">
+            <img class="language-image" src="img/about_page/html.png" />
+            <div class="language-name">HTML</div>
+        </div>
+        <div class="language-box">
+            <img class="language-image" src="img/about_page/css.png" />
+            <div class="language-name">CSS</div>
+        </div>
+    </div>
+     <li class="framework__container">   
+              <ul>
+                <li>
+                   <p class="Portfolio_About_d_1">Frameworks</p> 
+                   
+                </li>
+              </ul>
+              </li>
+            <li class="framework__container">
+              <ul>
+                <li>
+                  <p class="Portfolio_About_d_1">Skills</p>
+                  <img class="framework__arr" src="img/about_page/arr_r.svg">
+                </li>
+              </ul>
+            </li></div>
+        </section>
+         <section class="contact__container_me">  
```

Comment:
- [ ] Kindly ensure your contact section matches the given design as this is a requirement. Your design should match the layout, colors, images, fonts and text.

Your design
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/2c39bc93-90d3-4025-a9fc-ce3c8fa15daa)

Given design
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/e3b84c02-59b6-4826-a459-7d1c017e0af4)